assets/i18n/en_US.json

```diff
@@ -5,7 +5,7 @@
     "enabledLabel": "Enabled",
     "disabledLabel": "Disabled",
     "installed":"Installed: {version}",
-    "suggested":"Suggested: {version}",
+    "suggested":"Suggested: ",
```

Comment:
Did you remove `{version}` by mistake? or intentionally

Reply:
Intentionally, as it is no longer required anywhere.